kie-dmn-feel/pom.xml

```diff
@@ -71,6 +71,26 @@
 
   <build>
     <plugins>
+    
+        <plugin>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <configuration>
+            <fork>true</fork>
+            <meminitial>128m</meminitial>
+            <maxmem>512m</maxmem>
+            <showDeprecation>true</showDeprecation>
+            <showWarnings>true</showWarnings>
+            <source>1.8</source>
+            <target>1.8</target>
+            <testSource>1.8</testSource>
+            <testTarget>1.8</testTarget>
```

Comment:
@psiroky FYI this is just a draft PR for now, as a basis in order to discuss with @etirelli, but to answer your question I need the `-parameters` compile argument (JDK 8). Ref: https://github.com/droolsjbpm/kie-dmn/pull/21#discussion_r90006214

Not sure yet if it's to be added here or do we want to add it somewhere at higher scope?